docs/documentation/components/spot-small.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Spot small</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link href="../../static/css/manon.css" rel="stylesheet" />
+    <link href="../../../manon/scss/form/form-input-color-filled.css" rel="stylesheet" />
+    <link href="../../static/img/favicon.ico" rel="shortcut icon" />
+    <script defer src="../../static/js/manon.js"></script>
+  </head>
+  <body id="index" class="documentation">
+    <header>
+      <a href="#main-content" class="button focus-only skip-to-content">Ga direct naar inhoud</a>
+
+      <nav aria-label="Hoofdnavigatie">
+        <div>
+          <ul>
+            <li><a href="../../index.html">Home</a></li>
+            <li><a href="../components.html">Componenten</a></li>
+            <li><a href="../documentation.html">Documentatie</a></li>
+          </ul>
+        </div>
+      </nav>
+
+      <nav class="breadcrumb-bar">
+        <div>
+          <ul>
+            <li><a href="../components.html">Componenten</a></li>
+            <li><a href="./branding-colors.html">Kleurensets</a></li>
+            <li><a href="./accordion.html" aria-current="page">Spot small</a></li>
```

Comment:
```suggestion
            <li><a href="./spot-small.html" aria-current="page">Spot small</a></li>
```

Reply:
Is het een idee om deze voor consistentie met "finn klein" ook "spot klein" te noemen? (Sorry als dit een hele domme vraag is, ik ben niet bekend met de termen finn en spot.)

Reply:
Dat had er ook moeten staan. Bedankt voor het spotten. Ik heb ze aangepast. 

Ter info: Spot en Finn zijn verzonnen namen voor de kleurensets.